### PR TITLE
Added doc on how to configure CoreOS components to use HTTPS connecti…

### PR DESCRIPTION
**Pull request by kayrus**
…on with etcd cluster
/cc @joshix @omkensey

### CODE REVIEW

**Comment by joshix** [comment]
cf. https://www.digitalocean.com/community/tutorials/how-to-secure-your-coreos-cluster-with-tls-ssl-and-firewall-rules

**Comment by kayrus** [comment]
This doc explains how to configure TLS from the scratch, but it doesn't provide info on how to upgrade your existing cluster.

**Comment by brianredbeard** [comment]
@joshix any more thoughts on this?  As @kayrus mentioned the scope is different from the D.O. doc.

**Comment by joshix** [comment]
@kayrus @brianredbeard I was only storing the DO link here as a note to myself; It wasn't intended to be comparative, just conferred with. :)

File: etcd/configure_coreos_components_to_use_tls_etcd.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Configure CoreOS Components to Use TLS etcd
```
Comment (joshix) [inline etcd/configure_coreos_components_to_use_tls_etcd.md]: `# Configure CoreOS Components to Connect to etcd with TLS`

File: etcd/configure_coreos_components_to_use_tls_etcd.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+If you already have finished [HTTP to HTTPS migration][etcd-live-http-https] and would like to switch all your CoreOS components to use secure HTTPS connection with etcd cluster please follow instruction below.
```
Comment (joshix) [inline etcd/configure_coreos_components_to_use_tls_etcd.md]: This document explains how to configure all CoreOS components to use secure HTTPS connections to an etcd cluster. The target etcd cluster must already be using HTTPS for its own communications, as explained in the [etcd HTTP to HTTPS migration guide](etcd-live-http-https.md).

File: etcd/configure_coreos_components_to_use_tls_etcd.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+Main CoreOS components which use etcd are:
```
Comment (joshix) [inline etcd/configure_coreos_components_to_use_tls_etcd.md]: The main CoreOS components that use etcd are:

File: etcd/configure_coreos_components_to_use_tls_etcd.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+* [flannel][flannel]
+* [fleet][fleet]
+* [locksmith][locksmith]
```
Comment (joshix) [inline etcd/configure_coreos_components_to_use_tls_etcd.md]: locksmith doc directory URL is not local to coreos/docs/

**Comment by kayrus** [comment]
@joshix fixed

File: etcd/configure_coreos_components_to_connect_to_etcd_with_tls.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
+# Configure CoreOS Components to Connect to etcd with TLS
+
+This document explains how to configure all CoreOS components to use secure HTTPS connections to an etcd cluster. The target etcd cluster must already be using HTTPS for its own communications, as explained in the [tcd HTTP to HTTPS migration guide][tcd-live-http-https]
```
Comment (joshix) [inline etcd/configure_coreos_components_to_connect_to_etcd_with_tls.md]: s/tcd/etcd/

(typo in link)

**Comment by kayrus** [comment]
@joshix fixed

**Pull request merged**